high_card.rb

```diff
@@ -1,5 +1,70 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+require "pry"
+# Change ranks, suits, and players to constants
+RANKS = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"] #changed the order to reflect aces high
+SUITS = [ "hearts", "spades", "clubs", "diamonds" ]
+PLAYERS = []
+DECK = []
```

Comment:
Given that you're modifying this variable in `build_deck`, is it appropriate to declare it as a constant?